resources/ext/build-scripts/install-vendored-gems.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
```

Comment:
This is straight-up copy paste from googling "how to find the directory of the currently executing bash script".